burstphoto/exposure/exposure.swift

```diff
@@ -28,20 +28,23 @@ func correct_exposure(_ final_texture: MTLTexture, _ white_level: Int, _ black_l
             }
         }
         
-        var black_levels_mean = Array(black_level[exp_idx].map{Double($0)})
+        var black_levels_mean: [Double]
 
         // if exposure levels are uniform, calculate mean value of all exposures
         if uniform_exposure {
-            for i in 0..<black_level.count {
-                for j in 0..<black_levels_mean.count {
-                    black_levels_mean[j] += Double(black_level[i][j])
+            black_levels_mean = Array(repeating: 0.0, count: black_level[exp_idx].count)
+            for img_idx in 0..<black_level.count {
+                for channel_idx in 0..<black_levels_mean.count {
+                    black_levels_mean[channel_idx] += Double(black_level[channel_idx][img_idx])
```

Comment:
During testing, I found that channel_idx and img_idx have to be switched. Otherwise, I get an error.

Reply:
Yup, silly mistake. Should have tested after re-naming.